package.json

```diff
@@ -25,6 +25,7 @@
     "test:audit": "yarn audit",
     "test:integration": "rm -rf test/coverage && NODE_ENV=development nyc mocha -r dotenv/config  \"src/**/*-integration.test.ts\"",
     "test:unit": "rm -rf test/coverage && NODE_ENV=development nyc mocha --exclude \"src/**/*-integration.test.ts\"  \"test/unit/**/*.test.ts\" --recursive \"src/**/*.test.ts\"",
+    "test:u1": "rm -rf test/coverage && NODE_ENV=development nyc mocha --exclude \"src/**/*-integration.test.ts\" --recursive \"src/**/your-test*.test.ts\"",
```

Comment:
what's this for?

Reply:
That's an additional task for when you only want to run one unit test rather than the whole lot, thought it might be useful rather than changing the existing ones to do the same thing.

Reply:
Thanks for asking James and for the response Daniel, I didn't know what this was for either.